src/track/cue.h

```diff
@@ -47,12 +61,15 @@ class Cue : public QObject {
     QColor getColor() const;
     void setColor(QColor color);
 
+    CuePosition getCuePosition() const;
```

Comment:
get/setPosition(). Try to avoid repeating the redundant contextual information.

Reply:
Do we actually need the legacy set/getPosition() functions? In other parts of the code you have replaced `double` with `CuePosition`. Wouldn't it be consequent to use `CuePosition` everywhere? It is at least type safe and an improvement.